api/lib/infrastructure/repositories/certification-course-repository.js

```diff
@@ -16,7 +16,8 @@ function _toDomain(model) {
     lastName: model.get('lastName'),
     birthplace: model.get('birthplace'),
     birthdate: model.get('birthdate'),
-    rejectionReason: model.get('rejectionReason')
+    rejectionReason: model.get('rejectionReason'),
```

Comment:
Add a TODO: 1088 - code to fix
To modify the rejectionReason (not in certification but in assessment-result)